src/Miew.js

```diff
@@ -2941,9 +2942,9 @@ Miew.prototype.setPivotResidue = function (residue) {
     return;
   }
 
-  const pos = this._gfx.pivot.position;
+  const center = new THREE.Vector3();
```

Comment:
Here is the same point about vector creation. But I agree the modification of the temporary variable is safer than a change of pivot's component.

Reply:
done